### ISSUE TITLE
Add papers to the model list

### ISSUE THREAD

**Issue by zhujiem** [openbenchmark/BARS#1]
+ [TKDD'2020] Core Interest Network for Click-Through Rate Prediction
+ [CIKM2020] Deep Multi-Interest Network for Click-through Rate Prediction
+ [CIKM'21] Efficient Learning to Learn a Robust CTR Model for Web-scale Online Sponsored Search Advertising

+ [CIKM'21] Enhancing Explicit and Implicit Feature Interactions via Information Sharing for Parallel Deep CTR Models

+ [CIKM'21] One Model to Serve All: Star Topology Adaptive Recommender for Multi-Domain CTR Prediction

+ [CIKM'21] AutoIAS: Automatic Integrated Architecture Searcher For Click-Trough Rate Prediction

+ [CIKM'21] Click-Through Rate Prediction with Multi-Modal Hypergraphs

+ [CIKM'21] AutoHERI: Automated Hierarchical Representation Integration for Post-Click Conversion Rate Estimation
+ [CIKM'21] Disentangled Self-Attentive Neural Networks for Click-Through Rate Prediction

**Comment by zhujiem** [comment]
+ [SIGIR'2021] Explicit Semantic Cross Feature Learning via Pre-trained Graph Neural Networks for CTR Prediction
+ [KDD'2021] Dual Attentive Sequential Learning for Cross-Domain Click-Through Rate Prediction
+ [KDD'2021] Dual Graph Enhanced Embedding Neural Network for CTR Prediction
+ XCrossNet: Feature Structure-Oriented Learning for Click-Through Rate Prediction
+ Feature Interaction based Neural Network for Click-Through Rate Prediction
+ Learning Graph Meta Embeddings for Cold-Start Ads in Click-Through Rate Prediction
+ A Non-sequential Approach to Deep User Interest Model for CTR Prediction
+ AdnFM: An Attentive DenseNet based Factorization Machine for CTR Prediction
+ Feature Interaction based Neural Network for Click-Through Rate Prediction
+ Field-Embedded Factorization Machines for Click-through Rate Prediction
+ Robust Factorization Machines for User Response Prediction
+ Field-wise Learning for Multi-field Categorical Data
+ Feature Interaction based Neural Network for Click-Through Rate Prediction
+ An Efficient Deep Interaction Network for Click-Through Rate Prediction
+ Click-Through Rate Prediction Combining Mutual Information Feature Weighting and Feature Interaction
+ GateNet: Gating-Enhanced Deep Network for Click-Through Rate Prediction
+ Empirically Testing Deep and Shallow Ranking Models for Click-Through Rate (CTR) Prediction
+ Click-Through Rate Prediction Using Graph Neural Networks and Online Learning
+ Iterative Boosting Deep Neural Networks for Predicting Click-Through Rate
+ A New Click-Through Rates Prediction Model Based on Deep&Cross Network
+ User Response Prediction in Online Advertising
+ Evaluating deep learning based models for predicting click through rate
+ AMER: Automatic Behavior Modeling and Interaction Exploration in Recommender System
+ Predicting Response in Mobile Advertising with Hierarchical Importance-Aware Factorization Machine
+ Delayed Feedback Modeling for the Entire Space Conversion Rate Prediction
+ Feature Interaction Interpretability: A Case for Explaining Ad-Recommendation Systems via Neural Interaction Detection
+ Towards Automated Neural Interaction Discovery for Click-Through Rate Prediction
+ Correct Normalization Matters: Understanding the Effect of Normalization On Deep Neural Network Models For Click-Through Rate Prediction
+ DeepEnFM: Deep neural networks with Encoder enhanced Factorization Machine 
+ Factorization Machines with Regularization for Sparse Feature Interactions
+ Dual-attentional Factorization-Machines based Neural Network for User Response Prediction
+ [IJCAI'2020] A Dual Input-aware Factorization Machine for CTR Prediction
+ A Dynamic Neural Network Model for Click-Through Rate Prediction in Real-Time Bidding
+ [SEKE'20] Deep Graph Attention Neural Network for Click-Through Rate Prediction

+ [KDD'20] AutoFIS: Automatic Feature Interaction Selection in Factorization Models for Click-Through Rate Prediction

+ [CIKM'19] Regularized Adversarial Sampling and Deep Time-aware Attention for Click-Through Rate Prediction

+ [WWW'20] Field-aware Calibration: A Simple and Empirically Strong Method for Reliable Probabilistic Predictions


+ [WWW'20] Adversarial Multimodal Representation Learning for Click-Through Rate Prediction


+ [SIGIR'2020] AutoGroup: Automatic Feature Grouping for Modelling Explicit High-Order Feature Interactions in CTR Prediction


+ [IEEE Access'2019] Field-Aware Neural Factorization Machine for Click-Through Rate Prediction

+ [] Structured Semantic Model Supported Deep Neural Network for Click-Through Rate Prediction


+ [AAAI'19] Interaction-Aware Factorization Machines for Recommender Systems

+ Deep Neural Network-Based Click-Through Rate Prediction using Multimodal Features of Online Banners

+ [SIGIR'18] Combined Regression and Tripletwise Learning for Conversion Rate Prediction in Real-Time Bidding Advertising
+ [SIGIRW'18] Visualizing and Understanding Deep Neural Networks in CTR Prediction
+ [KDD'20] Towards Automated Neural Interaction Discovery for Click-Through Rate Prediction

**Comment by zhujiem** [comment]
+ [Arxiv'2021] FINT: Field-aware Interaction Neural Network For CTR Prediction
